handbook/people-ops/expenses.md

```diff
@@ -28,9 +28,20 @@ We use Expensify for expense report submission and approval. You will receive an
 - Add a description of your expense in the designated field. 
 
 
-### Expense categories
+### When submitting an expense
 
-Please ensure that each expense is properly categorized. Some frequently asked questions about categories include:
+- Please ensure that each expense is properly categorized. Some frequently asked questions about categories include:
+  - Software purchases or subscriptions should be categorized as "Software and SaaS <$1000"
+  - Computer purchases and other hardware (monitor, headphones, etc.) should be categorized as "Computer & Related <$1000" 
```

Comment:
As worded, this makes me believe that headphones/monitor are not part of the home office setup budget (even though it is stated otherwise elsewhere).

Reply:
The reason it's like this is because we don't have a "home office setup" category in Expensify, and that's because accounting wants to have specific categories for the items being expensed. So even though we have one home office setup budget, it can span across multiple categories.

Reply:
Can we clarify that here? I think it's useful information.

Reply:
Done!